Action for MISS_HIT fails on pull request: `miss_hit.cfg: error: config file contains errors`

The job fails for both the quality and style checking actions:

Here's the error report:

```
Run mh_metric --ci
miss_hit.cfg: error: config file contains errors
In miss_hit.cfg, line 8
| exclude_dir: "lib"
|              ^^^^^ error: does not exist
.: error: cannot find project root because the config file contains errors: please add a config file with the 'project_root' directive
MISS_HIT Metric Summary: 2 file(s) analysed, 3 error(s)
Error: Process completed with exit code 1.
```

@Remi-Gau is the `lib` directory perhaps a remnant from the use of this config in the `CPP_SPM` repo?

--- a/miss_hit.cfg
+++ b/miss_hit.cfg
@@ -5,7 +5,6 @@ regex_function_name: "fmrwhy(_[a-zA-Z]+)*"
 regex_script_name: "[a-z0-9]+(_[a-z0-9]+)*"
 
 exclude_dir: "dependencies"
-exclude_dir: "lib"
 
 suppress_rule: "copyright_notice"
 # copyright_entity: "CPP_SPM developers"

